usaspending_api/api_contracts/contracts/v2/reporting/agencies/agency_code/overview.md

```diff
@@ -63,7 +63,6 @@ This endpoint returns an overview of government agency submission data.
                         "tas_account_discrepancies_totals": {
                             "gtas_obligation_total": 66432,
                             "tas_accounts_total": 2342,
-                            "tas_obligation_not_in_gtas_total": 11543,
```

Comment:
What prompted this field to be removed? Isn't it necessary for the table?

![image](https://user-images.githubusercontent.com/22781949/101940038-1ac8b000-3ba3-11eb-98f9-1de6922e00ee.png)

Reply:
I asked the frontend devs about it, I believe we thought the reporting difference would be the difference between the `tas_accounts_total` and the `gtas_obligation_total`, since the ticket only mentioned the `missing_tas_accounts_counts` as needing the reporting agency differences tables,  but if that's incorrect I can add it back in.

Reply:
I'm not 100% sure that is true. For some reason I thought value in this table would be the magnitude of the differences (the addition of the absolute values of reporting differences per missing TAS), not a straight-up difference of one total - another total.

Another reason for the field was to make the API behavior and sorting more intuitive. Kinda awkward to test if the sorting works from the API response if the value needs to be calculated from two others

Reply:
That's fair, I've added it back in